graph_coloring.py

```diff
@@ -12,78 +12,104 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import networkx as nx
-from dimod import DiscreteQuadraticModel
-from dwave.system import LeapHybridDQMSampler
 import matplotlib
+import networkx as nx
+from dimod import CQM, BinaryQuadraticModel, Binary, quicksum
```

Comment:
Suggest using the full name:
```suggestion
from dimod import ConstrainedQuadraticModel, BinaryQuadraticModel, Binary, quicksum
```